addon/serializers/github-repository.js

```diff
@@ -7,6 +7,13 @@ export default GithubSerializer.extend({
       fullName: hash.full_name,
       name: hash.name,
       description: hash.description,
+      url: hash.html_url,
+      language: hash.language,
+      isFork: hash.fork,
+      isPrivate: hash.private,
```

Comment:
Would you mind renaming `url`, `isFork`, and `isPrivate` to `htmlUrl`, `fork`, and `private`.  I do like your naming conventions better but I'd just like to keep the naming as close to GitHub's as possible.